o-clock.cabal

```diff
@@ -28,6 +28,7 @@ library
                          Time.Rational
                          Time.TimeStamp
                          Time.Units
+  other-modules:       Time.Bool
```

Comment:
Actually it's better to put all modules into `exposed-modules`. Practice shows that in reality there's no actual benefit in putting something into `other-modules` but some drawbacks are possible (bad linking, absence of documentation). So currently the practice in Haskell community is to put every module into `exposed-modules` and if you want to specify the fact that module is internal you use `Internal` word here, like, `Time.Internal.Bool`. Though I'm not sure that this module should have name `Time.Internal.Bool`. It contains `==%` useful type family people might want to use in their applications.